protocols/indexer/contracts/Indexer.sol

```diff
@@ -193,27 +193,29 @@ contract Indexer is IIndexer, Ownable {
     *
     * @param _signerToken address
     * @param _senderToken address
-    * @param _count uint256
+    * @param _startAddress address The address to start from in the list of intents
+    * @param _count uint256 The total number of intents to return
     * @return locators address[]
     */
   function getIntents(
     address _signerToken,
     address _senderToken,
+    address _startAddress,
     uint256 _count
   ) external view returns (
     bytes32[] memory locators
   ) {
-
     // Ensure neither token is blacklisted.
     if (!blacklist[_signerToken] && !blacklist[_senderToken]) {
 
       // Ensure the index exists.
       if (indexes[_signerToken][_senderToken] != Index(0)) {
 
         // Return an array of locators for the index.
-        return indexes[_signerToken][_senderToken].fetchLocators(_count);
+        return indexes[_signerToken][_senderToken].fetchLocators(_startAddress, _count);
 
       }
+
```

Comment:
Can we clean this method up a bit?

My biggest complaint is the nesting of conditionals. 
We can pull both conditions out and check each one individually before reaching the end where it will return a valid case.

Reply:
I'll just combine into one if statement

Reply:
https://github.com/airswap/airswap-protocols/pull/183/commits/e6fb1a996d227315bd190ae365225d1b934bce85